aten/src/ATen/core/dispatch/Dispatcher.h

```diff
@@ -137,6 +137,10 @@ class CAFFE2_API Dispatcher final {
   // Invoke an operator via the boxed calling convention using an IValue stack
   void callBoxed(const OperatorHandle& op, Stack* stack) const;
 
+  // Retrieve a reference to the function for an operator.  This remains
+  // valid for as long as the underlying function si valid.
+  const KernelFunction& getKernel(const OperatorHandle& op, DispatchKey dispatchKey) const;
```

Comment:
This is a pretty significant addition to the API, given its potential for (mis)use in devirtualization-type use cases. Is it worth adding a warning of some sort? If anything I think the current comment kind of underplays the trouble you could get into. (also "si")

Reply:
*warning in the comments

Reply:
I'm dumping this, I can't use it due to compiler bug, see https://github.com/pytorch/pytorch/issues/48763